### ISSUE TITLE
a11y: Color Contrast - Search

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13735]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF 
Ratio: 1.78:1
Main Content
- Type to search for a resource…"

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT. No notes. Original comment:

> The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

Issue 131
